js/ui/speed_dial_action/speed_dial_main_item.js

```diff
@@ -8,6 +8,7 @@ import SpeedDialItem from "./speed_dial_item";
 import themes from "../themes";
 
 const FAB_MAIN_CLASS = "dx-fa-button-main";
+const FAB_MAIN_CLASS_HAS_TEXT = "dx-fa-button-has-label";
```

Comment:
I don't remember if we use verbs in class names.
```suggestion
const FAB_MAIN_CLASS_HAS_TEXT = "dx-fa-button-with-label";
```

Reply:
https://github.com/DevExpress/DevExtreme/blob/3f6e5b98c8c9e6192581a2ed56b3e081adb79bc0/styles/widgets/material/toolbar.material.less#L180 dx-button-has-text for example

Reply:
💩